pom.xml

```diff
@@ -198,6 +198,13 @@
         <module>testing/trino-tests</module>
     </modules>
 
+    <repositories>
```

Comment:
We don't want to add a new repository definition in the root POM, as it will be used for all modules. Ideally, we wouldn't have this at all, but at least it's currently limited to a few modules.